Source/Blazorise.Material/MaterialClassProvider.cs

```diff
@@ -37,5 +37,9 @@ public class MaterialClassProvider : Bootstrap.BootstrapClassProvider
 
     public override string BarItemHasDropdown( BarMode mode ) => mode == Blazorise.BarMode.Horizontal ? "dropdown" : null;
 
+    public override string AccordionToggle() => "btn btn-link btn-block text-left";
+
+    public override string AccordionToggleCollapsed( bool collapsed ) => collapsed ? null : "collapsed";
```

Comment:
is this right? If collapsed null otherwise "collapsed"?

Reply:
It is. We send it a `Visible`  parameter, and in here we name it `collapsed`. It's inverted so there's a confusion.